src/main/resources/archetype-resources/README.md

```diff
@@ -1,59 +1,141 @@
 3. Run `build.sh` file on your terminal
 4. Open file `EZYFOX_SERVER_HOME/settings/ezy-settings.xml` and add to `<zones>` tag:
+
 ```xml
-    <zone>
-		<name>${artifactId}</name>
-		<config-file>${artifactId}-zone-settings.xml</config-file>
-		<active>true</active>
-	</zone>
+<zone>
+	<name>${artifactId}</name>
+	<config-file>${artifactId}-zone-settings.xml</config-file>
+	<active>true</active>
+</zone>
 ```
-5. Run `console.sh` in `EZYFOX_SERVER_HOME` on your termial, if you want to run `ezyfox-server` in backgroud you will need run `start-server.sh` on your terminal
 
-# Deploy mapping
-Modules after will deploy to `ezyfox-server` will be mapped like this:
+5. Run `console.sh` in `EZYFOX_SERVER_HOME` on your terminal, if you want to run `ezyfox-server` in backgroud you will need to run `start-server.sh` on your terminal
+
+${symbol_pound}${symbol_pound} Run without ezyfox-server
+
+To run without `ezyfox-server` you need follow by steps:
+
+1. Run `bash export.sh` command
+2. Move to `${artifactId}-startup/deploy` folder
+
+${symbol_pound}${symbol_pound}${symbol_pound} On Windows
+
+You just need run `console.bat`
+
+${symbol_pound}${symbol_pound}${symbol_pound} On Linux
+
+1. To run to debug, you just need run `bash console.sh` on your terminal
+2. To run in background, you just need run `bash start-service.sh` on your terminal
+3. To stop your service, you just need run `bash stop-service.sh` on your terminal
+
+${symbol_pound}${symbol_pound} Run with specific configuration profile
+
+You can [read this guide](https://youngmonkeys.org/ezyfox-server-project-configuration/) to know how to run `ezyfox-server` or your application with a specific profile
+
+${symbol_pound} How to deploy?
+
+${symbol_pound}${symbol_pound} Deploy mapping
+
+Modules which are deployed to ezyfox-server will be mapped as follows::
+
 1. ${artifactId}-app-api => `ezyfox-server/apps/common/${artifactId}-app-api-${version}.jar`
 2. ${artifactId}-app-entry => `ezyfox-server/apps/entries/${artifactId}-app`
 3. ${artifactId}-common => `ezyfox-server/common/ ${artifactId}-common-${version}.jar`
 4. ${artifactId}-plugin => `ezyfox-server/plugins/${artifactId}-plugin`
 
-# How to test?
+${symbol_pound}${symbol_pound} Deploy with tools
+
+You can use bellow tools to copy jar files (follow by above mapping)
+
+1. [filezilla](https://filezilla-project.org/)
+2. [transmit](https://panic.com/transmit/)
+
+${symbol_pound}${symbol_pound} Deploy with scp
+
+We've already prepared for you `deploy.sh` file, you just need:
+
+1. Open `deploy.sh` file
+2. Set `ezyfoxServerLocal` by your `ezyfox-server` folder path on local
+3. Set `ezyfoxServerRemote` by your `ezyfox-server` folder path on remote
+4. Set `sshCredential` by your ssh credential, i.e `root@your_host.com`
+5. Run `bash deploy.sh` command
+6. After the deployment is done, you need open `settings/ezy-settings.xml` file in `ezyfox-server` on remote and add (if you have already done this step in the past, please skip it):
```

Comment:
Final comment: Need open --> need to open

Reply:
không cần final đâu em ạ, còn chưa ổn chỗ nào thì sửa chỗ đó em ạ =))

Reply:
@vu-luong review lại giúp anh nhé